plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -18,14 +18,16 @@ Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.2.6",
  org.eclipse.ui.ide,
  org.eclipse.jst.j2ee.web,
  com.google.cloud.tools.eclipse.sdk;bundle-version="0.1.0",
- com.google.cloud.tools.eclipse.sdk.ui;bundle-version="0.1.0"
+ com.google.cloud.tools.eclipse.sdk.ui;bundle-version="0.1.0",
+ org.eclipse.m2e.maven.runtime
```

Comment:
It would be nice if we could avoid introducing m2e dependencies into this bundle.  Or at least tie this together with `ServletClasspathContainer`.